### DISCUSSION TITLE
Relative Color Syntax from Stage 5

### DISCUSSION THREAD

**Discussion by adamharshbarger** [csstools/postcss-plugins#626] [Q&A]
Is there a feature of Preset-Env equivalent to the Relative Color Syntax from Stage 5? I see Safari has an implementation so far and I have not seen anything in the docs that is similar (I may have missed it). If not, will that be implemented soon. 

CSS Color Module 5 gives an example:

rgb(from  indianred 255 g b)

This takes the sRGB value of indianred (205 92 92) and replaces the red channel with 255 to give  rgb(255 92 92)

**Comment by romainmenke** [comment]
Hi @adamharshbarger,

Thank you for reaching out.

Currently there is no plugin in `postcss-preset-env` but we have a tracking issue for the request here : https://github.com/csstools/postcss-plugins/issues/177

We will start on this after we finish up `postcss-preset-env` `v8.0.0` : https://github.com/csstools/postcss-plugins/milestone/6.

**Reply by adamharshbarger** [reply]
Thank you for taking the time to answer this.
